### PR TITLE
initial wiring

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: pkg/mesh-networking/run.go [hunk 1]
```diff
@@ -85,3 +88,15 @@ func startComponents(meshNetworkingContext wire.MeshNetworkingContext) func(cont
 		return nil
 	}
 }
+
+func startTrafficPolicyReconciler(ctx context.Context, meshNetworkingContext wire.MeshNetworkingContext) {
```
Comment (harveyxia) [inline pkg/mesh-networking/run.go:92]: I would leave another comment here marking this as a temporary approach

File: pkg/mesh-networking/traffic-policy-temp/translation/framework/snapshot/reconcilers/destination_rule.go [hunk 2]
```diff
@@ -68,7 +69,7 @@ func (v *destinationRuleReconcilerBuilder) Build() (DestinationRuleReconciler, e
 		return nil, eris.New("Must either configure this reconciler to be cluster-scoped or explicitly scope it to a namespace/label")
 	}
 
-	if v.clusterScoped && (v.namespace != "" || len(v.labels) > 0) {
+	if v.clusterScoped && v.namespace != "" {
 		return nil, eris.New("Cannot be cluster-scoped and scoped to a namespace/label")
```
Comment (harveyxia) [inline pkg/mesh-networking/traffic-policy-temp/translation/framework/snapshot/reconcilers/destination_rule.go:73]:
```suggestion
		return nil, eris.New("Cannot be cluster-scoped and scoped to a namespace")
```

Comment (harveyxia) [reply pkg/mesh-networking/traffic-policy-temp/translation/framework/snapshot/reconcilers/destination_rule.go:73]: Also the top-level comment in this file needs to be updated, specifically the line `// a reconciler can either be whole-cluster scoped or scoped to a namespace and/or labels`

File: pkg/mesh-networking/run.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ func Run(ctx context.Context) {
 
 	// build all the objects needed for multicluster operations
 	meshNetworkingContext, err := wire.InitializeMeshNetworking(
-		contextutils.WithLogger(ctx, "access_control_enforcer"),
+		ctx,
```
[the remark this answers is not in the document]
Comment (yuval-k) [reply pkg/mesh-networking/run.go:21]: yea it got me confused..

**Pull request merged**